### PR TITLE
Increase a test timeout

### PR DESCRIPTION
**Pull request by nilsvu**
## Proposed changes

I've seen this fail macOS CI runs occasionally.

### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

**Review by nilsdeppe — APPROVED** [review]

File: tests/Unit/Domain/Structure/Test_OrientationMapHelpers.cpp [hunk 1]
```diff
@@ -593,6 +593,7 @@ void test_2d_orient_variables_on_slice() {
 
 }  // namespace
 
+// [[Timeout, 10]]
```
Comment (nilsdeppe) [inline tests/Unit/Domain/Structure/Test_OrientationMapHelpers.cpp]: I think this needs to be `TimeOut`

**Review by nilsdeppe — CHANGES_REQUESTED** [review]

File: tests/Unit/Domain/Structure/Test_OrientationMapHelpers.cpp (hunk 1 above)
Comment (nilsdeppe) [reply tests/Unit/Domain/Structure/Test_OrientationMapHelpers.cpp]: Wait, unclear... Documentation and code differ :P

Comment (nilsvu) [reply tests/Unit/Domain/Structure/Test_OrientationMapHelpers.cpp]: I copied this from somewhere, so it might be case insensitive

Comment (nilsvu) [reply tests/Unit/Domain/Structure/Test_OrientationMapHelpers.cpp]: Added a commit that just makes these consistent 🤷‍♂️

**Review by nilsdeppe — APPROVED** [review]

File: tests/Unit/Domain/Structure/Test_OrientationMapHelpers.cpp (hunk 1 above)
Comment (nilsdeppe) [reply tests/Unit/Domain/Structure/Test_OrientationMapHelpers.cpp]: Yea, at some point we did the smart thing and made it case insensitive :shrug:

**Pull request merged**